charts/rmd/values.yaml

```diff
@@ -0,0 +1,114 @@
+# Default values for rmd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+rmd:
+  # rmd.replicaCount -- (int) Number of Rmd replicas
+  replicaCount: 1
+  image:
+    # rmd.image.repository -- Rmd image name
+    repository: wiktorkepczynski/my_rmd
+     # Specify a tag to override which version of rmd to deploy.
+     # If no tag is specified the appVersion from Chart.yaml is used as tag.
+     # tag:
+    # rmd.image.pullPolicy -- Rmd image pull policy
+    pullPolicy: IfNotPresent
+  service:
+    # Specifiy secret for authentication.
+    # If not random token will be generated
+    # secret:
+    # rmd.service.name -- Rmd service name
+    name: rmd
+    # rmd.service.type -- Rmd service type
+    type: ClusterIP
+    # rmd.service.externalPort -- External Port of rmd service
+    externalPort: 80
+    # rmd.service.internalPort -- Internal Port of rmd service
+    internalPort: 80
```

Comment:
Can this be a high port like `8080` so the container doesn't need root or and caps?